compiler/luci-interpreter/src/kernels/Add.cpp

```diff
@@ -38,8 +38,10 @@ Add::Add(const Tensor *input1, const Tensor *input2, Tensor *output, const AddPa
 void Add::configure()
 {
   LUCI_INTERPRETER_CHECK(input1()->element_type() == input2()->element_type());
+  LUCI_INTERPRETER_CHECK(input1()->element_type() == output()->element_type());
   if (input1()->element_type() == DataType::S16)
   {
+    LUCI_INTERPRETER_CHECK(input1()->zero_points().size() == 1);
     LUCI_INTERPRETER_CHECK(input1()->zero_point() == 0 && input2()->zero_point() == 0 &&
                            output()->zero_point() == 0);
   }
```

Comment:
Maybe also add checking zero_points size for the input2:
```suggestion
  if (input1()->element_type() == DataType::S16)
  {
    LUCI_INTERPRETER_CHECK(input1()->zero_points().size() == 1 && input2()->zero_points().size() == 1);
    LUCI_INTERPRETER_CHECK(input1()->zero_point() == 0 && input2()->zero_point() == 0 &&
                           output()->zero_point() == 0);
  }
```